DSCResources/MSFT_xIPAddress/MSFT_xIPAddress.psm1

```diff
@@ -31,9 +29,9 @@ function Get-TargetResource
         [String]$AddressFamily = "IPv4"
     )
 
-
     $returnValue = @{
-        IPAddress = [System.String]::Join(", ",(Get-NetIPAddress -InterfaceAlias $InterfaceAlias -AddressFamily $AddressFamily).IPAddress)
+        IPAddress = [System.String]::Join(", ",(Get-NetIPAddress -InterfaceAlias $InterfaceAlias `
```

Comment:
I'm really unclear why this requests an IP address then doesn't use it for the Get-NetIPAddress call. This could return an array of IP addresses when its unclear that what we expect or not.